### ISSUE TITLE
Reproducible build process for cpt-bin on Windows 

### ISSUE THREAD

**Issue by aaron-kaplan** [iri-pycpt/pycpt#40]
The process for building linux and macos packages of cpt-bin is entirely within conda-build, but the process for Windows currently involves some manual setup steps, because not all of the build tools are packaged for conda. Need to automate those steps somehow. Create the missing conda packages? Script the creation of a properly configured Windows VM?

**Comment by aaron-kaplan** [comment]
Notes on the Windows build situation, which seems to have improved since I last looked:

https://github.com/conda-forge/conda-forge.github.io/issues/1891#issuecomment-2576385135
https://github.com/conda-forge/emacs-feedstock/issues/54#issuecomment-2576374780

**Comment by owinebar** [comment]
See https://github.com/conda-forge/emacs-feedstock/pull/96 for build bat and sed script you can use to run a build.sh script with the m2-* tools in your Windows build environment.  You'll have to put any Windows specific configuration in the build.sh (alongside Linux and OS X cases).  Also, I haven't worked out how to handle testing, if there's a command list that uses environment variables.  Those will probably have to be put in a separate shell script, with another batch file analagous to bld.bat.

Basically, the cygwin library is allergic to having more than one copy of itself on the path (regardless of version), which is antithetical to how conda works in general.

**Comment by owinebar** [comment]
Ok, see https://github.com/conda-forge/staged-recipes/pull/28882 for general-purpose scripts you will be able to put in your build/test environment.
